OpenVPNServiceRunner.cs

```diff
@@ -0,0 +1,181 @@
+﻿using Microsoft.Win32;
+using OpenVpn;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace OpenVpn
+{
+    class OpenVPNServiceRunner
```

Comment:
Are there functional changes hidden in here besides the stuff related to the new Log() method?